service/src/main/kotlin/fi/espoo/evaka/application/PendingDecisionEmailService.kt

```diff
@@ -0,0 +1,200 @@
+// SPDX-FileCopyrightText: 2017-2021 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+package fi.espoo.evaka.application
+
+import fi.espoo.evaka.daycare.domain.Language
+import fi.espoo.evaka.decision.getDecision
+import fi.espoo.evaka.emailclient.IEmailClient
+import fi.espoo.evaka.pis.getPersonById
+import fi.espoo.evaka.shared.async.AsyncJobRunner
+import fi.espoo.evaka.shared.async.SendPendingDecisionEmail
+import fi.espoo.evaka.shared.db.Database
+import mu.KotlinLogging
+import org.jdbi.v3.core.kotlin.mapTo
+import org.springframework.core.env.Environment
+import org.springframework.stereotype.Service
+import java.time.Instant
+import java.util.UUID
+
+private val logger = KotlinLogging.logger { }
+
+@Service
+class PendingDecisionEmailService(
+    private val asyncJobRunner: AsyncJobRunner,
+    private val emailClient: IEmailClient,
+    private val env: Environment
+) {
+    init {
+        asyncJobRunner.sendPendingDecisionEmail = ::doSendPendingDecisionsEmail
+    }
+
+    val fromAddress = env.getProperty("mail_reply_to_address", "no-reply.evaka@espoo.fi")
```

Comment:
You could just use `getRequiredProperty` and drop the default or drop the default config from `application.yaml`. It's now duplicated for no reason.